introducing-versioning-schemes.rst

```diff
@@ -0,0 +1,51 @@
+.. post:: November 28, 2023
+   :tags: docs, versions
+   :category: Changelog
+   :author: Santos
+   :location: CUE
+
+Introducing versioning schemes
+==============================
```

Comment:
```suggestion
Introducing versioning schemes
==============================
```
```suggestion
Introducing support for version-only projects
=============================================
```

Reply:
what about "Introducing support for projects with multiple versions without translations"? It's a long title, not sure which one is better p:

Reply:
I think we can keep it as suggested -- it's the thing that's changing, so seems good to highlight. 👍